proximal/examples/test_deconv_sv_psf.py

```diff
@@ -61,26 +61,26 @@ def weighted_planewise_conv(G, W, im_shape, n_psf: int):
     weight = W[:n_psf].reshape((n_psf, *im.shape))
 
     def forward(im_src, im_sim):
-        assert np.all(im_src.shape == im_sim.shape[1:])
-        assert im_sim.shape[0] == n_psf
+        assert np.all(im_src.shape == im_sim.shape)
 
+        im_sim[:] = 0
         for i in range(n_psf):
-            im_sim[i, :] = fftconvolve(psf_mode[i],
-                                       im_src * weight[i],
-                                       mode='same')
+            im_sim[:] += fftconvolve(psf_mode[i],
+                                     im_src * weight[i],
+                                     mode='same')
```

Comment:
> I see that you have changed the logic of the forward and adjoint functions and instead of summing the convolutions in the forward function, you are appending them to a list and then adding inside the adjoint function. Since I didn't understand this change, so thought of asking you if this could be breaking the code? I tried debugging but haven't been able to resolve it. Were you able to run it successfully on your end?

Yes my bad. Reverting the forward and adjoint operators back to `+=` . Please try it out again.